### PR TITLE
add README.md

### PR DESCRIPTION
**Pull request by cheewaio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,79 @@
...
+
+- Red Hat Enterprise Linux CoreOS
+
+## Operator versions
+
+- 3.5.0
+
+## Prerequisites
+
+1. Red Hat OpenShift Container Plataform 4.x installed
```
Comment (bitscuit) [inline README.md]: aren't operators also supposed to support OCP 3.11

File: README.md [hunk 2]
```diff
@@ -1,2 +1,79 @@
-# ibm-cloudctl-operator
-This operator provides IBM command line interface to manage cloud resources
+# ibm-platform-api-operator
```
Comment (bitscuit) [inline README.md]: maybe change to `IBM Platform API Operator` since this is a heading

File: README.md (hunk 1 above)
Comment (cheewaio) [reply README.md]: No, operators do not really work on OCP 3.x as there is no Operator Hub. 
For CP4Data, we made a special tar bar build for OCP 3.x with only a few operators

File: README.md [hunk 3]
```diff
@@ -1,2 +1,79 @@
-# ibm-cloudctl-operator
-This operator provides IBM command line interface to manage cloud resources
+# IBM Platform API Operator
+
+Operator used to manager IBM Platform API service, a REST service for viewing information about your cluster, manage your cluster, install Helm charts, download CLI tools, and much more.
```
Comment (bitscuit) [inline README.md]: `used to manager` -> `used to manage`

Comment (cheewaio) [reply README.md]: doh!

**Pull request merged**